testinfra/modules/socket.py

```diff
@@ -68,6 +68,9 @@ class Socket(Module):
     """Test listening tcp/udp and unix sockets
 
     ``socketspec`` must be specified as ``<protocol>://<host>:<port>``
+    
+    This module needs the  ``netstat`` binary to be installed in the
+    target environment otherwise it will not funciton.
```

Comment:
s/funciton/function

But I'd reword like this "This module require the ``netstat`` command on target host".